src/clients/evolutionManager.ts

```diff
@@ -0,0 +1,35 @@
+import { InstanceOptions, IOContext, AppGraphQLClient } from '@vtex/api'
+import { GraphQlError } from '../errors'
+
+export class EvolutionManager extends AppGraphQLClient {
+  constructor(context: IOContext, options: InstanceOptions) {
+    super('vtex.evolution-manager-graphql@0.x', context, options)
+  }
+
+  public saveWorkspacePromotion = (user: string, workspace: string): Promise<boolean> =>
```

Comment:
_nit_: Using arrow function in this case is unnecessary
```suggestion
  public saveWorkspacePromotion(user: string, workspace: string): Promise<boolean>
```